docs/reset-security-verification-for-wog-account.md

```diff
@@ -1,23 +1,23 @@
 # Reset MFA for WOG account
 
-This article guides you how to reset your WOG(SGGovt M365) MFA.
+This article provides instructions on how users can reset their WOG MFA(Multi-Factor Authentication). The WOG profile is listed as SGGovt M365 on the authenticator app. 
```

Comment:
Multi-Factor Authentication (MFA). "The WOG profile is listed as SGGovt M365 on the authenticator app." sorry i dont understand

Reply:
rephrased it.